cypress/integration/inventory/keyboard-shortcuts.spec.js

```diff
@@ -17,18 +24,40 @@ describe('ui-inventory: keyboard shortcut', () => {
     cy.visit(TopMenu.inventoryPath);
   });
 
-  after('Delete all data', () => {
+  afterEach('Delete all data', () => {
     cy.deleteUser(userId);
   });
 
   it('C345297 Keyboard Shortcut. Access to drop down menu', { tags: [testTypes.smoke] }, () => {
-    InventoryMainButton.verifyInventoryDropdownIsShown('false');
+    InventoryKeyboardShortcuts.verifyInventoryDropdownIsShown('false');
 
-    InventoryMainButton.openInventoryMenu();
-    InventoryMainButton.verifyInventoryDropdownIsShown('true');
-    InventoryMainButton.openShortcuts();
+    InventoryKeyboardShortcuts.openInventoryMenu();
+    InventoryKeyboardShortcuts.verifyInventoryDropdownIsShown('true');
+    InventoryKeyboardShortcuts.openShortcuts();
 
-    InventoryMainButton.waitModalLoading('Keyboard shortcuts');
-    InventoryMainButton.verifyInventoryDropdownIsShown('false');
+    InventoryKeyboardShortcuts.waitModalLoading('Keyboard shortcuts');
+    InventoryKeyboardShortcuts.verifyInventoryDropdownIsShown('false');
+  });
+
+  it('C345298 Keyboard Shortcut. Test the functionality of the different shortcut keys', { tags: [testTypes.smoke] }, () => {
```

Comment:
can't see this step from TC "Now search for any given instance. View the instance in the 3rd pane.
Then open the keyboard modal (step 1) and view the keyboard shortcut options. Close the modal.
Now test making a copy of the instance, by using keyboard shortcuts: Option + C (and on a Mac it is Alt + C)"

Reply:
Done